build/parents/build/pom.xml

```diff
@@ -0,0 +1,1522 @@
+
+        <!-- Test dependencies -->
+
+        <!-- >>> Common -->
+        <version.log4j>2.20.0</version.log4j>
+        <version.junit>4.13.2</version.junit>
+        <version.org.osgi.core>6.0.0</version.org.osgi.core>
+
+        <version.org.hamcrest>2.2</version.org.hamcrest>
+        <version.org.mockito>5.4.0</version.org.mockito>
+        <version.org.assertj.assertj-core>3.24.2</version.org.assertj.assertj-core>
+        <version.org.awaitily>4.2.0</version.org.awaitily>
+        <version.org.skyscreamer.jsonassert>1.5.1</version.org.skyscreamer.jsonassert>
+        <version.io.takari.junit>1.2.7</version.io.takari.junit>
+        <version.com.github.tomakehurst.wiremock>3.0.0-beta-10</version.com.github.tomakehurst.wiremock>
+        <version.org.apache.commons.lang3>3.13.0</version.org.apache.commons.lang3>
+        <version.org.apache.commons.math3>3.6.1</version.org.apache.commons.math3>
+        <version.org.apache.avro>1.11.2</version.org.apache.avro>
+        <version.org.jboss.weld>5.1.1.Final</version.org.jboss.weld>
+
+        <!-- >>> JDBC Drivers -->
+        <version.com.h2database>2.2.220</version.com.h2database>
+        <!-- Sticking to Derby 10.15 for now since later versions require JDK 17+, and we need to test with JDK 11.
+             See https://db.apache.org/derby/derby_downloads.html -->
+        <version.org.apache.derby>10.15.2.0</version.org.apache.derby>
+        <version.org.postgresql>42.6.0</version.org.postgresql>
+        <version.org.mariadb.jdbc>3.1.4</version.org.mariadb.jdbc>
+        <version.mysql.mysql-connector-java>8.1.0</version.mysql.mysql-connector-java>
+        <version.com.ibm.db2.jcc>11.5.8.0</version.com.ibm.db2.jcc>
+        <version.com.oracle.database.jdbc>23.2.0.0</version.com.oracle.database.jdbc>
+        <version.com.microsoft.sqlserver.mssql-jdbc>12.2.0.jre11</version.com.microsoft.sqlserver.mssql-jdbc>
+
+        <!-- >>> Performance tests -->
+        <version.org.openjdk.jmh>1.36</version.org.openjdk.jmh>
+
+        <!-- >>> JSR 352: JBatch runtime -->
+        <version.com.ibm.jbatch>2.1.1</version.com.ibm.jbatch>
+
+        <!-- >>> JSR 352: JBeret SE dependencies -->
+        <version.org.jboss.marshalling>2.1.1.Final</version.org.jboss.marshalling>
+        <version.org.wildfly.security.wildfly-security-manager>1.1.2.Final</version.org.wildfly.security.wildfly-security-manager>
+        <version.org.google.guava>31.1-jre</version.org.google.guava>
+
+        <!-- >>> Spring integration tests -->
+        <version.org.springframework.boot>3.1.1</version.org.springframework.boot>
+        <!-- Spring Boot 3 Atomikos starter and related libs version: -->
+        <version.com.atomikos>6.0.0</version.com.atomikos>
+
+        <!-- Public modules must be signed. -->
+        <release.gpg.signing.skip>false</release.gpg.signing.skip>
```

Comment:
```suggestion
        <!-- Only public modules must be signed. -->
        <release.gpg.signing.skip>false</release.gpg.signing.skip>
```

We'll probably want to move this to the root POM when we start flattening POMs and no longer publish the root POM.